apps/dashboard/src/components/Navbar.tsx

```diff
@@ -0,0 +1,76 @@
+
+import Image from 'next/image'
+import Link from 'next/link'
+import { GiHamburgerMenu } from 'react-icons/gi'
+import userLogo from '../utils/user.png'
+import NavbarItems from './NavbarItems'
+const Navbar = () => {
```

Comment:
It's good to be mindful of things like spacing so that the code is more readable. Things like adding a space between the imports and the main code or just removing extra lines like at the beginning of the file.